schema/SystemStructureCommon.xsd

```diff
@@ -409,23 +488,23 @@
                     <xs:documentation xml:lang="en">
                         This element provides for a transformation of integer parameter values
                         based on a mapping table and is valid for parameters of integer and 
-                        enumeration type.  Each mapping table entry is provided by a MapEntry
+                        enumeration types.  Each mapping table entry is provided by a MapEntry
```

Comment:
nitpick: In the spec we use the snippet "of all the integer types and Enumeration type" and here "of integer and enumeration types". I think we should decide which formulation to use and use the same for both places.